### PR TITLE
Add social index to article

### PR DESCRIPTION
**Pull request by jonathan-best**
* Add social shares index to the article ES query
* Add social shares to unit tests
* Update calculateIndicies to take an index param
* Add table to article for social interactions break down
* Add single metric to article for social interruptions

### CODE REVIEW

**Comment by jonathan-best** [comment]
screen shot 2015-10-20 at 15 51 22

File: src/server/queries/Social.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import assert from 'assert';
+import moment from 'moment';
+
+export default function PageViewsQuery(query) {
```
Comment (Lucas-Gluchowski) [inline src/server/queries/Social.js]: I recommend to name it PageEventsQuery as it is querying events index

File: src/server/esClient.js [hunk 2]
```diff
@@ -125,6 +131,23 @@ function retrieveMetaData(queryData){
   })
 }
 
+function retrieveSocialData(queryData){
+  return new Promise((resolve, reject) => {
+    let queryObject = SocialQuery(queryData);
+    let request = {
+      index: calculateIndices(queryData, process.env.ES_SOCIAL_INDEX_ROOT),
```
Comment (Lucas-Gluchowski) [inline src/server/esClient.js]: How about ES_EVENT_INDEX_ROOT. This index contains not only social data.

File: src/server/formatters/Articles.js [hunk 3]
```diff
@@ -6,7 +6,7 @@ export default function formatData(data) {
     assert.equal(Object.prototype.toString.call(data), '[object Array]',
       "argument 'data' should be an array");
 
-    assert.equal(data.length, 2,
+    assert.equal(data.length, 3,
       "argument 'data' should be an array of length 2");
```
Comment (peter-mouland) [inline src/server/formatters/Articles.js]: description should be updated too

File: test/formatters/Articles.spec.js [hunk 4]
```diff
@@ -41,6 +41,7 @@ describe('Article Formatter', function() {
     ArticleFormat(ESArticleResults)
       .then((data) => {
         expect(data.article).to.be.a.object;
+        console.log(data.article)
```
Comment (peter-mouland) [inline test/formatters/Articles.spec.js]: whoops

**Comment by peter-mouland** [comment]
cool

**Pull request merged**